packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -104,6 +139,67 @@ async function checkM365Account(): Promise<CheckFailure | undefined> {
   }
 }
 
+async function checkDependencies(
+  localEnvManager: LocalEnvManager,
+  depsChecker: VSCodeDepsChecker,
+  projectSettings: ProjectSettings
+): Promise<CheckFailure | undefined> {
+  try {
+    const deps = localEnvManager.getActiveDependencies(projectSettings);
+
+    const resolveRes = await depsChecker.resolve(deps);
```

Comment:
Is `resolve` fast-fail? Prefer to set `fastFail: false` to resolve deps as many as possible.

Reply:
Good idea, it has been changed.